From: Werner Koch <wk@gnupg.org>
Date: Wed, 5 Jul 2017 09:01:59 +0200
Subject: doc: Typo fix

--

(cherry picked from commit d2aa2204cc7fc6b481412ed8ba9f65dfc64c6627)
---
 src/init.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/init.c b/src/init.c
index a1ee505..b01e089 100644
--- a/src/init.c
+++ b/src/init.c
@@ -214,7 +214,7 @@ _gpg_err_set_errno (int err)
 
 
 
-/* Internal tracing functions.  Except for TARCE_FP we use flockfile
+/* Internal tracing functions.  Except for TRACE_FP we use flockfile
  * and funlockfile to protect their use. */
 static FILE *trace_fp;
 static int trace_save_errno;
